crt_portal/cts_forms/views.py

```diff
@@ -336,6 +336,103 @@ def serialize_data(report, request, report_id):
     return output
 
 
+class ProfileView(LoginRequiredMixin, FormView):
+    """Can be used for updating section filter for a profile"""
+    form_class = ProfileForm
+
+    def post(self, request):
+        """Update or create Profile"""
+        if request.user.profile:
```

Comment:
We'll need a more robust check here. As written, this will never evaluate to `False`. 

If a User has no associated profile, this will throw an `RelatedObjectDoesNotExist` exception and return a 500 response.